### DISCUSSION TITLE
download error

### DISCUSSION THREAD

**Discussion by mld68** [aclap-dev/video-downloadhelper#346] [Help and Q&A]
### Your question

Hi, over a month ago i purched the liscense to download videos to watch at home. today I must have clicked the wrong button because it no longer downloads it gives an error now. Is there anyway to reset it so i can start downloading again?

### Addon & CoApp version and details

Hi, over a month ago i purched the liscense to download videos to watch at home. today I must have clicked the wrong button because it no longer downloads it gives an error now. Is there anyway to reset it so i can start downloading again?

**Comment by Mattmjs** [comment]
You can reset everything to it's default in the Settings / behavior tab. There is also Reset settings under the More tab.
